src/EditorFeatures/Test2/Rename/InlineRenameTests.vb

```diff
@@ -58,6 +58,33 @@ Namespace Microsoft.CodeAnalysis.Editor.UnitTests.Rename
             End Using
         End Function
 
+        <WpfTheory>
+        <CombinatorialData, Trait(Traits.Feature, Traits.Features.Rename)>
```

Comment:
WorkItem attribute?

Reply:
Not really applicable here. The feature already works.